ql/termstructures/globalbootstrap.hpp

```diff
@@ -67,13 +76,15 @@ template <class Curve> class GlobalBootstrap {
                     AdditionalPenalties additionalPenalties,
```

Comment:
The comment above this constructor should probably be updated to cover your addition.  I'd also move it so that it's a comment for the class itself, not just this constructor (since it applies to at least another constructor).